mlora/model/LoraLiner.py

```diff
@@ -104,11 +104,13 @@ def replace_init_lora_tensor(lora: Lora, lora_a: torch.Tensor, lora_b: torch.Ten
     def forward(self, data: torch.Tensor, input_args: MultiLoraBatchData) -> torch.Tensor:
         # data shape is: batch_size * max_seq_len * dim
         # result = data @ self.weight_.transpose(0, 1)
-        result = self.weight_.forward(data)
+        with torch.cuda.nvtx.range("liner_base"):
+            result = self.weight_.forward(data)
 
         if not self.enable_lora_:
             return result
 
+        torch.cuda.nvtx.range_push("linear_lora")
```

Comment:
Better to profile each LoRA adapter separately?

Reply:
good idea, i will modify it